pkg/gateway/mirror/handler/grpc/mock_test.go

```diff
@@ -28,9 +27,9 @@ type gatewayMock struct {
 	ForwardedContextFunc     func(ctx context.Context, podName string) context.Context
 	FromForwardedContextFunc func(ctx context.Context) string
 	BroadCastFunc            func(ctx context.Context,
-		f func(ctx context.Context, target string, vc vald.ClientWithMirror, copts ...grpc.CallOption) error) error
+		f func(ctx context.Context, target string, vc service.MirrorClient, copts ...grpc.CallOption) error) error
 	DoMultiFunc func(ctx context.Context, targets []string,
-		f func(ctx context.Context, target string, vc vald.ClientWithMirror, copts ...grpc.CallOption) error) error
+		f func(ctx context.Context, target string, vc service.MirrorClient, copts ...grpc.CallOption) error) error
```

Comment:
Maybe `gatewayMock` should go to `internal/test/mock`?

Reply:
Thank you for your comment 🙏 
Since internal/test/mock is a mock library for internal libraries, I think there is no need to move it at this time. 🤔 
The implementation part of this mock depends only on the service layer and is currently an implementation that is not used elsewhere excluding the mirror gateway, so I think it is fine to leave it as it is.